src/metabase/models/audit_log.clj

```diff
@@ -79,17 +82,17 @@
                  :topic    unqualified-topic
                  :details  details
                  :model    model-name
-                 :model_id model-id)
+                 :model_id model-id
+                 :user_id  user-id)
      ;; TODO: temporarily double-writing to the `activity` table
```

Comment:
Minor thing: maybe can explicitly call out when to delete this? Eg. 
`;; TODO: once on MB 48, delete form that double writes to the activity table.`

Then, if it somehow gets missed (though removing the table entirely should cause this to be caught), whoever is in there can see that we expect to remove it and know roughly when that should happen.